R/pkg/R/DataFrame.R

```diff
@@ -2604,18 +2603,17 @@ setMethod("join",
               if (is.null(joinType)) {
                 sdf <- callJMethod(x@sdf, "join", y@sdf, joinExpr@jc)
               } else {
-                if (joinType %in% c("inner", "cross",
+                valid_join_types <- c("inner", "cross",
```

Comment:
nit: let's keep the naming rule here -> `validJoinTypes`

Reply:
ok